src/config/mod.rs

```diff
@@ -38,9 +38,16 @@ pub fn read_config(logger: &Logger) -> Result<Config, AppError> {
   if paths.projects.exists() {
     for maybe_project_file in WalkDir::new(&paths.projects).follow_links(true) {
       let project_file = maybe_project_file?;
-      if project_file.metadata()?.is_file() {
+      if project_file.metadata()?.is_file() && !project_file.file_name().to_os_string().eq(".DS_Store") {
         let raw_project = read_to_string(project_file.path())?;
-        let mut project: Project = toml::from_str(&raw_project)?;
+        let mut project: Project = match toml::from_str(&raw_project) {
+          Ok(x) => Ok(x),
+          Err(e) => {
+            eprintln!("There is an issue in your config for project {}", project_file.file_name().to_string_lossy());
+            Err(e)
```

Comment:
I'm don't have a compiler here but couldn't this be
```suggestion
          e @ Err(e) => {
            eprintln!("There is an issue in your config for project {}", project_file.file_name().to_string_lossy());
            e
```